IdentityCore/src/validation/MSIDAadAuthorityCache.m

```diff
@@ -164,6 +166,8 @@ - (BOOL)processImpl:(NSArray<NSDictionary *> *)metadata
             VERIFY_HOST_STRING(alias, @"aliases", YES);
         }
         
+        record.openIdConfigurationEndpoint = openIdConfigEndpoint;
```

Comment:
we never check that openIdConfigurationEndpoint is non-nil and NSString (at least I didn't find)

Reply:
done